index.bs

```diff
@@ -12171,7 +12173,7 @@ to Consider</a>
 	been demonstrated
 	<a href="https://pdfs.semanticscholar.org/cfd2/6a17234696593919df3f880a235d6ac5871d.pdf">
 	by Nakibly et. al. for HTML</a>. The <a href="https://w3c.github.io/hr-time/#sec-privacy-security">
-	Security & Privacy appending of High Resolution Time</a> should be consulted for further
+	Privacy & Security appendix of High Resolution Time</a> should be consulted for further
```

Comment:
The section title is "Security and Privacy Consideration", so we might want to flip the order.

Reply:
But the HR-time spec calls it "Privacy and Security" if you follow the link.  That's why I changed the order.

Reply:
ACK